javascript-if/main.js

```diff
@@ -0,0 +1,79 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number) {
+  if (number % 2) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person) {
+    return false;
+  }
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if ((pH < 7) && (pH > -1)) {
+    return 'acid';
```

Comment:
pH levels can be fractional numbers. What happens if `pH` is something like `-0.23`?

---

![image](https://user-images.githubusercontent.com/16143771/145304401-86c7f672-b081-4fef-a41a-280c7686afda.png)